### DISCUSSION TITLE
SemVer 2.0 support for the CHANGELOG

### DISCUSSION THREAD

**Discussion by cdwillie76** [Zuehlke/poinz#319] [Ideas]
I created a pull request to update the SEMVER regex pattern to support SerVer 2.0. This is my first attempt with code changes to a node app and open to learning how to improve it.  I split out the parse functions since I was getting a webpack error when I exported the other functions to support the unit tests.  Thanks!

https://github.com/Zuehlke/poinz/pull/318

**Comment by xeronimus** [comment]
thanks for the contribution. PR merged, see #321
